### PR TITLE
Assignment #1 Third Issue

### PR DESCRIPTION
**Pull request by pnotequalnp**
Randomize order of questions displayed each run and add unit test to ensure that none are null still.

### CODE REVIEW

**Comment by mmwoodfo** [comment]
I do not believe this bit of code here does what the project asks for? That is, it is not randomizing the order of the _answers_ for each question, but rather randomizing the order the _questions_ that are displayed.
```C#        
public static Question[] RandomizeQuestions(Question[] questions)
        {
            Random rnd = new Random();
            return questions.OrderBy(question => rnd.Next()).ToArray();
        }
```

> **Issue 3 (extra credit)** Feature request: Currently the application displays all of the answers in the same order each time. Adjust the application so the **_answers_ are displayed in a random order.** Add appropriate unit tests and ensure the app continues to function properly.

ps: I like your use of Linq's `OrderBy` method, however.

**Comment by pnotequalnp** [comment]
> I do not believe this bit of code here does what the project asks for? That is, it is not randomizing the order of the _answers_ for each question, but rather randomizing the order the _questions_ that are displayed.
> 
> ```cs
> public static Question[] RandomizeQuestions(Question[] questions)
>         {
>             Random rnd = new Random();
>             return questions.OrderBy(question => rnd.Next()).ToArray();
>         }
> ```
> 
> > **Issue 3 (extra credit)** Feature request: Currently the application displays all of the answers in the same order each time. Adjust the application so the **_answers_ are displayed in a random order.** Add appropriate unit tests and ensure the app continues to function properly.
> 
> ps: I like your use of Linq's `OrderBy` method, however.

Indeed I misread answers as questions.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -66,6 +67,20 @@ public static string GetFilePath()
             return "Trivia.txt";
         }
 
+        public static Question[] RandomizeAnswers(Question[] questions)
```
Comment (mmwoodfo) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs]: Looks good!

File: PrincessBrideTrivia/PrincessBrideTrivia/Question.cs [hunk 2]
```diff
@@ -5,5 +5,8 @@ public class Question
         public string Text;
         public string[] Answers;
         public string CorrectAnswerIndex;
+        public string CorrectAnswer {
```
Comment (breaman) [inline PrincessBrideTrivia/PrincessBrideTrivia/Question.cs]: Looks like you added a property purely for the purpose of testing which is not a good idea. If you would have used this in the program to say "Sorry, but the correct answer is..." that would have made sense, but it isn't considered good practice to add properties purely for testability.

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (breaman) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs]: Nice solution for randomizing the answers.
